source/models/row.py

```diff
@@ -0,0 +1,76 @@
+import os, json, uuid
+from outputs.exceptions import *
+from models.index import Index
+
+class Row:
```

Comment:
I reviewed this yesterday, why are u asking me to review this again ??

Reply:
the problem was because the rebase i think it is done now